src/app/(BottomTabNavigation)/Cases/CaseScreen/index.tsx

```diff
@@ -22,9 +31,15 @@ function CasesScreen() {
           <Text>Go Back</Text>
         </TouchableOpacity>
       </View>
+      {status === Eligibility.ELIGIBLE && (
```

Comment:
If eligible, the file claim and opt out buttons should lie below the `Case Status` bar and according to the Figma (new update), there should be a line with "OR" in between

Reply:
Make sure you import the line as an SVG from Figma when you do this instead of hard coding a line with text!